OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -161,100 +189,155 @@ open class OptimoveConfigBuilder: NSObject {
         return self
     }
 
-    /**
-     Internal SDK embedding API, do not call or depend on this method in your app
-     */
-    @discardableResult public func setBaseUrlMapping(baseUrlMap: ServiceUrlMap) -> OptimoveConfigBuilder {
```

Comment:
are we sure no hybrid sdk use this? @cgwyllie you probably remember what was this for?

Reply:
A good question, I'm not sure what it was used by, but certainly stuff we annotated as embedding API tends to be hybrid SDKs...

Looks like Unity needs it: https://github.com/search?q=org:optimove-tech++setBaseUrlMapping&type=code

Reply:
You're right -- this method was in use by other SDKs. Moving it back

Reply:
@vvoicehovics the ref in Unity SDK was only to iOS SDK xcframework. We can delete this API.

Reply:
turns used by in-app people, shall put back